### PR TITLE
Add USWDS typography to non-USWDS components and Storybook

### PR DESCRIPTION
**Pull request by jamigibbs**
## Chromatic

https://2900-component-typography--65a6e2ed2314f7b8f98609d8.chromatic.com

## Description

This will update Storybook and components to all display with the font normalization of Source Sans Pro.

Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2900
related https://github.com/department-of-veterans-affairs/vets-website/pull/30089

## QA Checklist
- [ ] Component maintains 1:1 parity with design mocks
- [ ] Text is consistent with what's been provided in the mocks
- [ ] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ ] Tab order and focus state work as expected
- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ ] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ ] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ export class VACrisisLineModal {
               </a>
             </li>
             <li>
-              <va-icon icon="deaf" class="va-clm__icon" size={3}></va-icon>
+              <va-icon icon="tty" class="va-clm__icon" size={3}></va-icon>
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx:128]: The new tty icon was missing so I'm adding it now:

Screenshot 2024-06-11 at 10 23 38 AM

File: packages/web-components/src/components/va-promo-banner/va-promo-banner.css [hunk 2]
```diff
@@ -19,8 +21,8 @@
 .va-promo-banner__icon {
   background-color: var(--vads-color-white);
   border-radius: 50%;
-  height: 4.2rem;
-  width: 4.2rem;
+  height: 42px;
+  width: 42px;
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-promo-banner/va-promo-banner.css:25]: I think it was a typo how the values were set here so this updates it to match what's in [Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=2941-2265&t=HzGcE4nkWLtVsfi1-0):

Screenshot 2024-06-11 at 10 25 42 AM

Screenshot 2024-06-11 at 10 25 48 AM

File: packages/web-components/src/components/va-table/va-table.tsx [hunk 3]
```diff
@@ -8,17 +8,17 @@ import {
...
 @Component({
   tag: 'va-table',
   styleUrl: 'va-table.scss',
   shadow: true,
 })
 export class VaTable {
+  /**
+  * This is a wrapper component whose job is to marshal the cells in slotted va-table-rows 
+  * through DOM manipulation for projection into the slots of the va-table-inner component for rendering
+  */
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-table/va-table.tsx:21]: This comment block was displaying on the Storybook page itself so I'm moving it into the class to stop that from happening.

Screenshot 2024-06-11 at 10 27 23 AM
Screenshot 2024-06-11 at 10 27 23 AM

**Review by Andrew565 — APPROVED** [review]

File: .github/workflows/publish-storybook.yml [hunk 4]
```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: main
+          fetch-depth: 0
```
Comment (jamigibbs) [inline .github/workflows/publish-storybook.yml:17]: This is to try and fix the deployment issue we're seeing from the Storybook 7 build that was merged: https://github.com/department-of-veterans-affairs/component-library/actions/runs/9470553487/job/26091744901

**Pull request merged**